dll/win32/browseui/internettoolbar.cpp

```diff
@@ -907,33 +923,17 @@ HRESULT STDMETHODCALLTYPE CInternetToolbar::ShowDW(BOOL fShow)
             return hResult;
     }
 
-#if 0 // Why should showing the IDockingWindow change all bands? Related to CORE-17236
-    if (fMenuBar)
-    {
-        hResult = IUnknown_ShowDW(fMenuBar, fShow);
-        if (FAILED_UNEXPECTEDLY(hResult))
-            return hResult;
-    }
-
-    if (fControlsBar)
+    // TODO: Why should showing the IDockingWindow change all bands? Related to CORE-17236
```

Comment:
Is this also now related to [CORE-19659](https://jira.reactos.org/browse/CORE-19659) ?